app/javascript/components/mentoring/mentor_conversations_list.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react'
+import { usePaginatedQuery } from 'react-query'
+import dayjs from 'dayjs'
+import RelativeTime from 'dayjs/plugin/relativeTime'
+import { Pagination } from './mentor_conversations_list/pagination'
+import { UrlParams } from '../../utils/url_params'
+dayjs.extend(RelativeTime)
```

Comment:
Is this something we can do globally or is it desirable to happen on a per component basis?

Reply:
We can do globally, but let's hold off until we have the need to do it.

Reply:
OK, fine.